Lab-Instructions/0.getting-started.md

```diff
@@ -1,36 +1,34 @@
-
 ## Introduction
-This workshop is designed to teach you how to add AI features, like AI Search and AI Chat, to an existing application while working with the GitHub Copilot for Azure agent for assistance. The lab exercise modules are designed to be completed in order (1, 2, 3, etc.) building up the application progressively through each exercise.
+This lab is designed to teach you how to add AI features, like AI Search and AI Chat, to an existing application hosted on Azure while working with the GitHub Copilot for Azure extension for assistance. The lab is 75 minutes long and the exercise modules are designed to be completed in order (1, 2, 3, etc.) building up the application progressively through each exercise.
 
-## Logistics
-The lab is 75 minutes. There are two presenters and four proctors, so you will have support throughout the lab. If you need help, please raise your hand and a proctor will explain how to complete these tasks. We're here to help!
+## Prerequisite
+- VS Code
+- GitHub Copilot Chat extension
+- GitHub Copilot for Azure extension
+- GitHub Copilot License
+- Azure Account
+- Azure Developer CLI
+- Docker Desktop
 
 ## Prerequisite Knowledge 
-We assume you know the basics of creating and editing files in Visual Studio Code. VS Code isn't very different from other code editors, but if you have never used any code editor you might need a little help.
+We assume you know the basics of creating and editing files in Visual Studio Code. VS Code isn't very different from other code editors, but if you have never used any code editor you might need a little help. To learn more about VS Code, visit [here](https://code.visualstudio.com/docs).
 
 ## Prepare environment
-In the lab environment, the [`lab-setup.ps1`](../lab-setup.ps1) script has already been run for you. This means:
-
-1. **VS Code should already open** with the lab repo loaded.
-2. **Docker Desktop should be running.**
-3. **All required tools and extensions should be up to date.**
-
-Proceed directly to [Authenticate to GitHub](#Authenticate-to-GitHub) below.
-
-If you do not see VS Code or Docker Desktop running, double-click the `lab-setup` script located on the desktop and wait for the script to finish (2-3 minutes):
-
-<img src="..\img\lab-setup.png" alt="lab-setup script" style="max-width: 50%; height: auto;">
-
-Alternatively, you can run the script from a PowerShell terminal:
+In the lab environment, the [`lab-setup.ps1`](../lab-setup.ps1) script is provided for you to prepare your environment. 
+You can run the script from a PowerShell terminal: 
 
 ```powershell
 irm https://raw.githubusercontent.com/microsoft/build25-LAB309/refs/heads/main/lab-setup.ps1 | iex
 ```
 
+Once you run the script, This means:
+
+1. **VS Code should already open** with the lab repo loaded.
+2. **Docker Desktop should be running.**
+3. **All required tools and extensions should be up to date.**
+
```

Comment:
Any thoughts on having just the manual setup instructions? The script is meant to be run in a controlled lab environment, and I don't think would run on non-Windows platforms

Reply:
I'll remove the script and the instructions. Thanks!